### ISSUE TITLE
[Bug]: Audio makes me sound like a squirrel and crackles

### ISSUE THREAD

**Issue by Shkeffy** [obs-ndi/obs-ndi#984]
### Operating System Version

Windows 11

### CPU

Ryzen 5 3600

### GPU

RTX 3060 ti

### OBS Version

30.0.2

### OBS Installation Method

.exe

### OBS-NDI Version

03:20:51

### OBS-NDI Installation Method

.exe

### OBS Log [URL]

https://obsproject.com/logs/FlpDJpX3zdt2dSqG

### NDI Version

5.6.1.0

### Describe the bug

So I use NDI from OBS, and I send that to Splitcam on the same PC. The only reason I'm doing that is I was testing it before I get my streaming laptop. Anyways, when I send it to Splitcam, and stream from Splitcam, the audio crackles and sounds like a squirrel. If you go to https://twitch.tv/shkeffy, you can hear it.

**Comment by Shkeffy** [comment]
It also happens when recording from Splitcam, so I really don’t know what it could be…

**Comment by paulpv** [comment]
Do you have a direct link to a video showing the problem (and timestamp if it isn't obvious)

**Comment by Shkeffy** [comment]
Uh it’s all throughout the twitch: https://twitch.tv/shkeffy

**Comment by paulpv** [comment]
By "sound like a squirrel" I was expecting a chipmunk type of sound.

I watched all of https://www.twitch.tv/videos/2075781328 and parts of a few others.
I don't know what your normal voice sounds like.
I am not saying it sounds "perfect", but I did not hear anything major bad/off with your voice being noticeably squirrelly or crackly.
Maybe if I viewed the waveform in an audio editor I might see some glitches, but TBH I'm not mentally prepared to spend the time to download your channel's A/V content and analyze its audio.

I have heard a lot worse coming from obs-ndi.
